lib/sycamore/sycamore/query/planner.py

```diff
@@ -129,6 +135,105 @@ def process_json_plan(
     return result_nodes[0], nodes
 
 
+def postprocess_llm_helper(user_message: str) -> str:
+    messages = [
+        {
+            "role": "system",
+            "content": """You are a helpful agent that assists in small transformations
+                    of input text as per the instructions. You should make minimal changes
+                    to the provided input and keep your response short""",
+        },
+        {"role": "user", "content": user_message},
+    ]
+
+    prompt_kwargs = {"messages": messages}
+    chat_completion = OpenAI(OpenAIModels.GPT_4O.value).generate(
```

Comment:
we'll want to use the llm object from the class here so we can support non-openai cases

Reply:
Fixed to use llm_client from the class.